cmd/global-query/pkg/distributed/query.go

```diff
@@ -37,6 +39,10 @@ func NewQueryRunner(resolver hosts.Resolver, querier Querier, opts ...QueryOptio
 	return
 }
 
+const (
+	unboundedQuery = "unbounded query"
```

Comment:
Nitpick: Mb rather `errUnboundedQuery`?

Reply:
It's not an error though. Just a re-usable string.